packages/flutter_tools/templates/app/web/index.html.tmpl

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang ="en">
```

Comment:
This isn't necessary. There's no text here -- so there shouldn't be a lang attribute. We should make sure the "locale" field is used to create a "lang" attribute when drawing Text nodes with drawParagraph, maybe, if we don't already do that.

Setting it to "en" unconditionally is _definitely_ wrong, since it would imply all Flutter apps are English.

Reply:
Yes I completely forgot about locale management...sorry